Symfony/CS/Tests/Fixer/Symfony/MethodSeparationFixerTest.php

```diff
@@ -0,0 +1,529 @@
+ */
+
+namespace Symfony\CS\Linter;
+
+/**
+ * Dummy linter. No linting is performed. No error is raised.
+ *
+ * @author Dariusz Rumiński <dariusz.ruminski@gmail.com>
+ *
+ * @internal
+ */
+final class NullLinter implements LinterInterface
+{
+    /**
+     * {@inheritdoc}
+     */
+    public function lintFile($path)
+    {
+        unset($path);
+    }
+
+    /**
+     * {@inheritdoc}
+     */
+    public function lintSource($source)
+    {
+        unset($source);
+    }
+}
+',
+        );
+
+        // do not touch anonymous functions (since PHP doesn't allow
+        // for class attributes being functions :(, we only have to test
+        // those used within methods)
+        $cases[] = array(
+            '<?php
+class MethodTestAnonymous
+{
+    public function method444a()
+    {
+        $text = "hellow";
+        $example = function ($arg) use ($message) {
+            var_dump($arg . " " . $message);
+        };
+        $example($text);
+        $example = function($arg) use ($message) {
+            var_dump($arg . " " . $message);
+        };
+        $example = function /*test*/ ($arg) use ($message) {
+            var_dump($arg . " " . $message);
+        };
+    }
+}',
+        );
+
+        $cases[] = array(
+            '<?php
+class MethodTest1
+{
+    private $c; //
+
+    public function method444a()
+    {
+    }
+
+    /**
+     *
+     */
+    public function method444b()
+    {
+    }
+
+    //
+
+    public function method444c()
+    {
+    }
+
+    private $a;
+
+    public function method444d()
+    {
+    }
+
+    private $b;
+    //
+
+    public function method444e()
+    {
+    }
+
+    public function method444f()
+    {
+    }
+
+    private $d; //
+
+    public function method444f1()
+    {
+    }
+
+    /**/
+
+    public function method444g()
+    {
+    }
+}',
+            '<?php
+class MethodTest1
+{
+    private $c; //
+    public function method444a()
+    {
+    }
+    /**
+     *
+     */
+    public function method444b()
+    {
+    }
+
+    //
+
+
+    public function method444c()
+    {
+    }
+
+    private $a;
+    public function method444d()
+    {
+    }
+    private $b;
+    //
+    public function method444e()
+    {
+    }
+
+    public function method444f()
+    {
+    }
+
+    private $d; //
+    public function method444f1()
+    {
+    }
+
+    /**/
+    public function method444g()
+    {
+    }
+}',
+        );
+
+        // spaces between methods
+        $cases[] = array(
+            '<?php
+abstract class MethodTest3
+{
+    public function method021()
+    {
+    }
+
+    public static function method121()
+    {
+    }
+
+    abstract protected function method221();	'.'
+
+    final private function method321a()
+    {
+    }
+}'
+        ,
+            '<?php
+abstract class MethodTest3
+{
+    public function method021()
+    {
+    }
+
+    public static function method121()
+    {
+    }
+
+
+    abstract protected function method221();
+
+
+
+	'.'
```

Comment:
I take this out, it is a little work around for when working with PHPStorm since it trims lines like this (which I consider a bug btw.)